src/webserver.go

```diff
@@ -179,14 +179,21 @@ func initDBconnection() {
 	}
 }
 
-func TeslaMateAPIHandleErrorResponse(c *gin.Context, s1 string, s2 string) {
-    log.Println("[error] " + s1 + " - " + c.Request.RequestURI + " error in execution! " + s2)
-    c.JSON(http.StatusOK, gin.H{"error": s2})
+func TeslaMateAPIHandleErrorResponse(c *gin.Context, s1 string, s2 string, s3 string) {
+	log.Println("[error] " + s1 + " - (" + c.Request.RequestURI + "). " + s2 + "; " + s3)
+	c.JSON(http.StatusOK, gin.H{"error": s2})
 }
 
 func TeslaMateAPIHandleSuccessResponse(c *gin.Context, s string, j JSONData) {
-    log.Println("[info] " + s + " - " + c.Request.RequestURI + " executed successful.")
-    c.JSON(http.StatusOK, j)
+	// print to log about request
+	if gin.IsDebugging() {
+		log.Println("[debug] " + s + " - (" + c.Request.RequestURI + ") returned data:")
+		js, _ := json.Marshal(jsonData)
```

Comment:
```suggestion
		js, _ := json.Marshal(j)
```
If I'm correct, the variable is called `j` and not `jsonData`